core/node/eth_sender/src/eth_tx_aggregator.rs

```diff
@@ -1147,6 +1158,32 @@ impl EthTxAggregator {
 
         GatewayMigrationState::from_sl_and_notification(self.settlement_layer, notification)
     }
+
+    async fn is_waiting_for_interop_roots(
+        &self,
+        storage: &mut Connection<'_, Core>,
+    ) -> Result<bool, EthSenderError> {
+        let latest_processed_l1_batch_number = storage
+            .interop_root_dal()
+            .get_latest_processed_interop_root_l1_batch_number()
+            .await?
+            .unwrap();
```

Comment:
Why unwrap? get_latest_processed_interop_root_l1_batch_number returns None if L1 batch is not yet sealed.